dotnet/DotnetModuleSample/DotnetModuleSample/DotnetModuleSample.csproj

```diff
@@ -37,8 +37,8 @@
     <StartProgram>$(SolutionDir)DotnetModuleSample\bin\Release\gw.exe</StartProgram>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Azure.IoT.Gateway, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
Maybe you can ignore the version number, culture and processorAchitecture here to save maintaining effort in the future.